File: 3. Python/1. Bronze/25304.py
# variables setting
x = int(input())
n = int(input())
total = 0

# sum
for _ in range(n):
    temp = list(map(int, input().split()))
    total += temp[0] * temp[1]

# print
if x == total:
    print("Yes")
else:
    print("No")

File: 1. C, C++/1. Bronze/25304.c
// 25304
#include <stdio.h>

int main() {
    int total, num, a, b;
    scanf("%d", &total);
    scanf("%d", &num);
    for(int i = 0; i < num; i++) {
        scanf("%d %d", &a, &b);
        total -= a * b;
    }
    if(total == 0) printf("Yes");
    else printf("No");
}